src/diagnostic.re

```diff
@@ -0,0 +1,71 @@
+exception Error(string);
+
+type diagnostic = {
+  source: string,
+  line: int,
+  column: int,
+  reason: string
+};
+
+let pp = ({line, column, reason, source}) => {
```

Comment:
We’re currently using Babel-code-frame to report the errors, it’s in lib/cli.js. It think you might be able to reuse a lot of that logic, and just return the source, line, and column range to the JS wrapper, then let that handle things.

Reply:
Ah, sorry I've missed such convenience function, and I'd like to use it, thanks 🙏

Reply:
Now, we'll use `babel-code-frame` for reporting parse error too :+1:

Reply:
As I actually wrote it, I included it in this PR, because the difference of code 
was smaller than I thought